src/clj/game/core/runs.clj

```diff
@@ -4,30 +4,224 @@
          gain-run-credits update-ice-in-server update-all-ice
          get-agenda-points gain-agenda-point optional-ability
          get-remote-names card-name can-access-loud can-steal?
-         prevent-jack-out card-flag? can-run?)
+         prevent-jack-out card-flag? can-run? run-approach-ice
+         jack-out successful-run end-run update-step corp-phase-53)
 
 ;;; Steps in the run sequence
+(defn approach-server
+  "[5]: the runner approaches the attacked server"
```

Comment:
Is there a specific reason for these being in reverse order?

Reply:
I borrowed this code from Neal and he had them in this order as they called each other directly. Could easily be reordered now.